src/command_modules/azure-cli-maps/azure/cli/command_modules/maps/_help.py

```diff
@@ -0,0 +1,62 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['maps'] = """
+    type: group
+    short-summary: Manage Azure Maps accounts.
```

Comment:
So should I remove the helps['maps'] or helps['maps account']?
I removed helps['maps'], because removing helps['maps account'] makes "az maps account -h" to not give the short summary.